app/scrapers/sachsen_scraper.rb

```diff
@@ -0,0 +1,187 @@
+    page.search('//td[@class="dxdvItem_EDAS"]/table')
+  end
+
+  def self.extract_title(item)
+    item.search('.//tr[3]/td/b').text.strip
+  end
+
+  def self.extract_meta_text(item)
+    text = item.search('.//tr[3]/td').text.split("\n").find{ |t| t.include?('KlAnfr') || t.include?('GrAnfr') }
+    text.strip if !text.nil?
+  end
+
+  def self.extractType(text)
+    if (text[0..5] == 'KlAnfr')
+      Paper::DOCTYPE_MINOR_INTERPELLATION
+    else
+      Paper::DOCTYPE_MAJOR_INTERPELLATION
+    end
+  end
+
+  def self.extract_date(meta_text)
+    Date.parse(meta_text.match(/\d+\.\d+\.\d{4}/).to_s)
+  end
+
+  def self.extract_reference(meta_text)
+    meta_text.split('Drs')[1].strip
+  end
+
+  def self.extract_originators(meta_text)
+    people = [meta_text.partition(/\p{Upper}{2}/)[0].strip]
+    parties = [meta_text.match(/\p{Upper}\p{Lower}*\p{Upper}+\s*\p{Upper}*/)[0].strip]
+    originators = { people: people, parties: parties}
+  end
+
+  def self.extract_overview_paper(item, doctype)
+    meta_text = extract_meta_text item
+    meta_text = meta_text[6..meta_text.length].strip
+    full_reference = extract_reference(meta_text)
+    {
+      legislative_term: full_reference.split('/')[0],
+      full_reference: full_reference,
+      reference: full_reference.split('/')[1],
+      doctype: doctype,
+      title: extract_title(item),
+      url: nil,
+      published_at: extract_date(meta_text),
+      originators: extract_originators(meta_text),
+      is_answer: nil,
+    }
+  end
+
+  def self.extract_detail_paper(item)
+    meta_text = extract_meta_text(item)
+    type = extractType(meta_text)
+    extract_overview_paper(item, type)
+  end
+
+  def self.switchToTerm(i, m, top)
+    nav = top.frame_with(name: 'head').click
+    nav = m.get("/redirect.aspx?wahlperiode=" + i.to_s)
+    nav.forms.first.submit
+  end
+
+  class Detail < DetailScraper
+    def scrape
+      m = mechanize
+      top = m.get BASE_URL
+      SachsenScraper.switchToTerm(@legislative_term, m, top)
+      content = top.frame_with(name: 'content').click
+      content = searchDetail(content)
+      paper = nil
+      SachsenScraper.extract_overview_items(content).each do |item|
+        paper = SachsenScraper.extract_detail_paper(item)
+        content = m.click(item.search('//a').first)
+        answeredDate = content.search('//*[@id="ctl00_masterContentCallback_content_tabTreffer_trefferDataView_IT0_HyperLink6"]').first
+        if (answeredDate.nil? || answeredDate.text.empty?)
+          paper[:is_answer] = false
+        else
+          paper[:is_answer] = true
+          paper[:published_at] = SachsenScraper.extract_date(answeredDate.text)
+          top = m.get(extract_viewer_url(content))
+          pdfurl = extract_pdf_url(top)
+          paper[:url] = pdfurl
+        end
+      end
+      paper
+    end
+
+    def extract_pdf_url(top)
+      nav = top.frame_with(name: 'navigation').click
+      onloadValue = nav.search("//body").first.attribute("onload")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Use snake_case for variable names.